.github/workflows/build.yml

```diff
@@ -77,8 +77,7 @@ jobs:
         if: matrix.target == 'darwin'
         shell: bash
         run: |
-          # install wails with amd64 arch, even for arm64, to be able to use it for the build
-          GOARCH=amd64 go install github.com/wailsapp/wails/v2/cmd/wails@v2.6.0
+          export PATH=$PATH:~/go/bin/darwin_amd64
```

Comment:
runner macos-latest is now macos 14 with M1 CPU, so arm64 arch. So to fix the build on amd, we add the go/bin/darwin_amd64 directory in the path